### DISCUSSION TITLE
Add sync methods

### DISCUSSION THREAD

**Discussion by Ilyazyk** [FreeOpcUa/opcua-asyncio#538] [Ideas]
Can we please have added two Client's sync methods:
asyncua\sync.py

class Client:
    @syncmethod
    def write_values(self):
        pass
    
    @syncmethod
    def read_values(self):
        pass

**Comment by swamper123** [comment]
You can provide a PR for that.

**Reply by Ilyazyk** [reply]
I just can't push new branch from my Eclipse (push not permitted) nether by https nor ssh. Any tips are welcome. Or just create the PR please.

**Reply by AndreasHeine** [reply]
fork, create branch, comit and pr ?

**Reply by Ilyazyk** [reply]
Finally, I've created PR "write_values & read_values #541". Hope it's done correctly.
